addons/mail/static/src/components/call_main_view/call_main_view.js

```diff
@@ -17,6 +17,18 @@ export class CallMainView extends Component {
         useComponentToModel({ fieldName: 'component' });
         useRefToModel({ fieldName: 'tileContainerRef', refName: 'tileContainer', });
         useUpdateToModel({ methodName: 'onComponentUpdate' });
+
+        onMounted(() => {
+            this.resizeObserver = new ResizeObserver(() => {
+                this.callMainView.onComponentUpdate();
+            });
+            this.resizeObserver.observe(this.root.el);
```

Comment:
I'm not sure how possible it is, but `this.root.el` might change during re-render? In that case this code should be moved to `onComponentUpdate` itself (mounted + patched) and not just mounted. (and then possibly have disconnect in will patch?)

Reply:
I don't know if it's possible either, buf if it's not it would be preferable to not reset the observer after each render

Reply:
It's possible to do more complicated and only remove/add new observer if the `.el` actually changes by saving the previous value and comparing, but if might indeed not be necessary at all, I'm not completely sure.

Reply:
I don't know owl well enough to know if there could be a corner case where it can happen, but it seems to conserve that same element for the whole lifespan of the component in that case